mycore-restapi/src/main/java/org/mycore/restapi/v2/MCRRestV2App.java

```diff
@@ -50,8 +51,11 @@
 
     public MCRRestV2App() {
         super();
+        if (MCRConfiguration2.getBoolean("MCR.RestAPI.V2.EnableContentNegotiationViaFileExtension").orElse(false)) {
```

Comment:
We should always enable it (can be excluded by regex: see above)

Reply:
done